media/Config/vscode.css

```diff
@@ -118,7 +118,7 @@ li {
   font-weight: bold;
 }
 
-#locaForSelect {
+#spanTagForSelect {
```

Comment:
Is this change related with the commit title, message?

Reply:
I don't think this change related with the commit title, message.
But request to change id for better understanding come from this pr.
make another pr or make commit for this change, which one is better idea?

Reply:
I add this change to commit message.